### DISCUSSION TITLE
Set size of Section title reported on top of each page

### DISCUSSION THREAD

**Discussion by sonarventu** [joaomlourenco/novathesis#444] [Q&A] [answered]
Hi! Thanks a lot for your great work, I love this template.
I have a little problem: some of the titles of my Sections are very long, and as a result, when they get reported on top of each "left" page, they occupy two lines (see picture).
Is there a way to control their font size?
Thanks!
Davide
![Screenshot from 2023-06-15 10-45-10](https://github.com/joaomlourenco/novathesis/assets/80964900/d5bbfecc-e160-4b35-bc83-104c542fb564)

**Comment by joaomlourenco** [comment]
Hi. Thanks for the feedback.  I assume you'r not in Portugal.  If yo need some assistance on adapting the template to your School, let me know.

Concerning your question, i do recommend you to use LaTeX's feature that applies to `\chapter`, `section`, `\label`, etc…
```latex
\section[abbreviated title for heading and table of contents]{Long title in the main document}.
```

**Reply by sonarventu** [reply]
Thanks a lot! 
You're right, I'm writing from Italy, but so far I more or less managed to adapt this wonderful template to my needs (the possibility to add one's own cover was very useful in this respect).

The solution you're proposing is quite convincing, but I still think that it would be important to have the actual title shown in full in the table of contents... Is there a way to control the two things separately? Thanks!
Davide

**Reply by joaomlourenco** [reply, marked as answer]
Add the following to your `Config/5_packages.tex`
```latex
\makeevenhead{novathesis@mainmatter}{\small\textsc{\leftmark}}{}{}
\makeoddhead{novathesis@mainmatter}{}{}{\small\textsc{\rightmark}}
```
and replace the `\small` with whatever font size you prefer.

**Reply by sonarventu** [reply]
Thanks, this completely solved the problem :)
